sku.go

```diff
@@ -572,3 +572,13 @@ func (s *SKU) Equal(other *SKU) bool {
 		localErr != nil &&
 		otherErr != nil
 }
+
+// Includes returns true if the SKU's name is in the list of SKUs.
+func (s *SKU) Includes(skuList []SKU) bool {
```

Comment:
The name of this function threw me, I expected `list.includes(item)`, but this is `item.includes(list)`, which is weird. Should this be `item.MemberOf(list)` or `item.IncludedIn(list)` or something?

Reply:
Same here, https://github.com/Azure/skewer/pull/24#discussion_r2072538478

Reply:
updated